scripts/pdpModule.js

```diff
@@ -0,0 +1,140 @@
+var pdpModule = (function () {
+    const categoriesItemsList = fetchCategoriesList();
+    const menuItemsList = fetchMenuList();
+    const cartItemsList = fetchCartList();
+    const itemsByCategoryMap = createItemsByCategoryMap(menuItemsList);
+
+    return {
+        init: display
+    };
+
+    function display() {
+        _displayCategories("recommended");
+        _displayMenu("recommended")
+        _displayCart();
+    }
+
+    function _displayCategories(category) {
+        const categoriesContainer = document.querySelector(".categories");
+        const categoriesItemsEl = document.createElement("ul");
+        categoriesItemsList.forEach(element => {
+            const categoriesItemEl = document.createElement("li");
```

Comment:
As discussed, avoid inline anonymous functions. They make the code hard to reason about and also violate SRP